_episodes/02-setup.md

```diff
@@ -39,6 +39,38 @@ which means that any changes pushed to
 another Git host server
 in a later lesson will include this information.
 
+> ## Line Endings
+>
+> As with other keys, when you hit the 'return' key on your keyboard, 
+> your computer is encodes this input. 
```

Comment:
I think yo have a typo: "is encodes" -> "encodes"

Reply:
Otherwise, it looks good.! Thanks @jimmyodonnell

Reply:
Good catch! Fixed.